so, the test fails with:

```
%MSG-w EvFDaqDirector:  PreGlobalEndLumi 18-Apr-2024 14:28:23 CEST  Run: 379416 Lumi: 97
 Handles to check for files to delete were not set by the input source...
%MSG
18-Apr-2024 14:28:23 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/user/cmsbuild/store/data/Run2024C/EphemeralHLTPhysics0/RAW/v1/000/379/416/00000/e8dd5e3c-216f-4545-acb6-ab86c9161085.root
Traceback (most recent call last):
  File "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_14_1_X_2024-04-18-1100/bin/el8_amd64_gcc12/convertToRaw", line 250, in <module>
    with open(eor_file) as f:
FileNotFoundError: [Errno 2] No such file or directory: '/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_14_1_X_2024-04-18-1100/unit_tests/testConvertToRaw/run379416_ls0097/run379416/run379416_ls0000_EoR.jsn'
```

whereas on a `lxplus` node it works fine. Does the problem lie in the fact that the bot is not allowed to create sub-folders? Is there a way to test in an environment similar to the PR tests?

--- a/EventFilter/Utilities/plugins/RawEventFileWriterForBU.cc
+++ b/EventFilter/Utilities/plugins/RawEventFileWriterForBU.cc
@@ -309,8 +309,8 @@ void RawEventFileWriterForBU::stop() {
 //TODO:get from DaqDirector !
 void RawEventFileWriterForBU::makeRunPrefix(std::string const& destinationDir) {
   //dirty hack: extract run number from destination directory
-  std::string::size_type pos = destinationDir.find("run");
-  std::string run = destinationDir.substr(pos + 3);
+  std::string::size_type pos = destinationDir.rfind("/run");
+  std::string run = destinationDir.substr(pos + 4);
   run_ = atoi(run.c_str());
   std::stringstream ss;
   ss << "run" << std::setfill('0') << std::setw(6) << run_;

--- a/HLTrigger/Tools/test/BuildFile.xml
+++ b/HLTrigger/Tools/test/BuildFile.xml
@@ -1,0 +1,2 @@
+<!-- tests convertToRaw utility script -->
+<test name="testConvertToRaw" command="testConvertToRaw.sh"/>

--- a/HLTrigger/Tools/test/testConvertToRaw.sh
+++ b/HLTrigger/Tools/test/testConvertToRaw.sh
@@ -1,0 +1,43 @@
+#!/bin/bash
+
+# Pass in name and status
+function die {
+  printf "\n%s: status %s\n" "$1" "$2"
+  exit $2
+}
+
+check_for_success() {
+    "${@}" && echo -e "\n ---> Passed test of '${@}'\n\n" || exit 1
+}
+
+check_for_failure() {
+    "${@}" && exit 1 || echo -e "\n ---> Passed test of '${@}'\n\n"
+}
+
+inputfile=/store/data/Run2024C/EphemeralHLTPhysics0/RAW/v1/000/379/416/00000/e8dd5e3c-216f-4545-acb6-ab86c9161085.root
+
+echo "========================================"
+echo "Testing convertToRaw in ${SCRAM_TEST_PATH}."
+echo "----------------------------------------"
+echo
+
+echo "========================================"
+echo "testing help function "
+echo "----------------------------------------"
+echo
+
+convertToRaw --help  || die "Failure running convertToRaw --help" $?
+
+echo "========================================"
+echo "testing successful conversion"
+echo "----------------------------------------"
+echo
+
+check_for_success convertToRaw -f 1 -l=1 -v $inputfile
+
+echo "========================================"
+echo "testing failing conversion"
+echo "----------------------------------------"
+echo
+
+check_for_failure convertToRaw -f 1 -l=-1 -s rawDataRepacker $inputfile

